tsdb/head.go

```diff
@@ -1537,29 +1537,34 @@ func (m seriesHashmap) del(hash uint64, lset labels.Labels) {
 	}
 }
 
+const (
+	defaultStripeSize = 1 << 14
```

Comment:
If we are allowing changing of stripe size, we may as well expose this too.

Reply:
Good point. Fixed.